ietf/meeting/models.py

```diff
@@ -746,6 +746,14 @@ def json_dict(self, host_scheme):
     def qs_assignments_with_sessions(self):
         return self.assignments.filter(session__isnull=False)
 
+    def qs_timeslots_in_use(self):
+        """Get QuerySet containing timeslots used by the schedule"""
+        return self.meeting.timeslot_set.filter(sessionassignments__schedule=self)
+
+    def qs_sessions_scheduled(self):
+        """Get QuerySet containing sessions assigned to timeslots by this schedule"""
+        return self.meeting.session_set.filter(timeslotassignments__schedule=self)
+
```

Comment:
There's no direct shorthand for the timeslots tied to a `Schedule` - the one on `Meeting` comes from the `ForeignKey` in `SchedTimeSessAssignment`. However, because each `Schedule` also has a `meeting`, it'd be better to rewrite these as `TimeSlot.objects.filter(sessionassignments__schedule=self)`. That will also catch the possible (though afaik unlikely) case of a `Schedule` with `meeting is None`.